### PR TITLE
Add filter and digits to AndesTextfield.

### PR DESCRIPTION
**Pull request by nesmarsollie**
Both fields are useful to customize input filter and allowed characters.

## Useful links
- [Contributing](https://github.com/mercadolibre/fury_andesui-android/blob/develop/CONTRIBUTING.md) has more information and tips for a great pull request. This will give you a nice introduction about how to contribute with Andes.
- [Wiki](https://github.com/mercadolibre/fury_andesui-android/wiki) has all the info you need to start with Andes. Even I heard there's a video there...

## Motivation
We need to customize filters and allowed digits. This solution is simple and flexible enough to apply it to any workaround. 

## Description
The primary use on this is to allow us to filter and mas numbers with custom DecimalFormatSymbols and with fixed amount of decimals.  

How this work: 
Assign the property textDigits programatically and will impact as digits xml label on edittext.
Assign the property textFilter and will apply the filter to the EditText. 

## Affected component
AndesTextfield

## Checks
Are you sure you followed all those steps? In such case, let's say with me "I solemnly declare that ...":
- [ ] I have coded an example inside the Demo App,
- [ ] I have added proper tests,
- [x] I have modified the [Changelog](https://github.com/mercadolibre/fury_andesui-android/blob/develop/CHANGELOG.md) file,
- [ ] I have updated GitHub wiki,
- [ ] I have the explicit approval of one or more members of the UX Team,
- [x] I have updated the version in gradle.properties file.

## Change type
This is easy, let us know what this code is about:
- [ ] This code adds a new component
- [x] This code includes improvements to an existent component
- [ ] This code improves or modifies ONLY the Demo App.

## Check common errors
Whether you are an Android or iOS developer and if you're still there then we'll give you a present:
https://proandroiddev.com/how-to-maximize-androids-ui-reusability-5-common-mistakes-cb2571216a9f

### CODE REVIEW

**Comment by furyGorgoryAdmin** [comment]
### Informative Menu <h2>  ### 

### Use any of these tags in a pull request description or leave a comment ### 
| Tag | Description |
|---|---|
| **[canary]**| Creates a build that can only be deployed with canary strategy. | 
| **[build]** |  Creates version or build from a branch that has been tagged as "no releasable". | 
| **[name:version-name]** |Specify the name of the build. For example: [build] [name:1.2.3-name] or [canary] [name:1.2.3-name].|
| **[major]** | To automatically increase the first position of the version. For example: [build] [major] or [canary] [major].|
| **[minor]** | To automatically increase the second position of the version. For example: [build] [minor] or [canary] [minor]. |
| **[patch]** | To automatically increase the third position of the version. For example: [build] [patch] or [canary] [patch].|  

[For more information about ci-cd](http://docs.furycloud.io/?lang=esp&section=businessassurance)
[For more information about semver](https://semver.org/#summary)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
-## v2.9.0
+# v2.9.0
```
Comment (barriosnahuel) [inline CHANGELOG.md:1]: sería la v2.10, lo actualizas por fa (junton con la property de version).

de paso no le pones el formato `Componente: Variant` o Feature que estas agregando así queda como el resto?

File: components/src/main/java/com/mercadolibre/android/andesui/textfield/AndesTextfield.kt [hunk 2]
```diff
@@ -149,10 +151,38 @@ class AndesTextfield : ConstraintLayout {
...
+    var textDigits: String? = null
+        set(value) {
+            field = value
+            textComponent.keyListener = DigitsKeyListener.getInstance(field)
+        }
+
+    /**
+     * Internal countFilter to limit field size
+     */
+    private var countFilter: InputFilter? = null
```
Comment (barriosnahuel) [inline components/src/main/java/com/mercadolibre/android/andesui/textfield/AndesTextfield.kt:180]: cuántos filters podríamos tener? quisieramos llevarlo a algún map o alguna otra estructura o no vale la pena?

Comment (nesmarsollie) [reply components/src/main/java/com/mercadolibre/android/andesui/textfield/AndesTextfield.kt:180]: El countFilter es algo muy puntual, lo veo muy bien asi.
El textFilter, bueno es algo discutible si queremos permitir varios o uno solo. Tener uno solo nos permite un poco mas de control. Igual se puede refactorizar a futuro agregando alguna opcion que permita definir varios sin perder compatibilidad.

File: CHANGELOG.md (hunk 1 above)
Comment (nesmarsollie) [reply CHANGELOG.md:1]: mmm. no te sigo, no sacamos nunca mas de una funcion por version ? O sea cada merge es una verison nueva ? o ya se hizo release de la 2.9.0 ?
Por otro lado no entiendo bien como es el formato Componente: Variant , por ejemplo en la 2.7.0 se puso un comentario muy similar al mio.

Comment (barriosnahuel) [reply CHANGELOG.md:1]: si perdon no aclare, es que ya salio la 2.9 por eso!
Lo del formato ponelo como acá:
image

que es el neuvo formato que queremos usar para aclarar quién contribuyo en cada caso

File: components/src/main/java/com/mercadolibre/android/andesui/textfield/AndesTextfield.kt (hunk 2 above)
Comment (barriosnahuel) [reply components/src/main/java/com/mercadolibre/android/andesui/textfield/AndesTextfield.kt:180]: joya, gracias!

**Pull request closed without merging**